frontend/packages/kubevirt-plugin/src/selectors/vm/flavor-data.ts

```diff
@@ -18,12 +18,9 @@ export const getFlavorText = ({
   memory: string;
   flavor?: string;
 }) => {
-  const vcpusCount = vCPUCount(cpu);
-  const vcpusText = `${vcpusCount} vCPU${vcpusCount > 1 ? 's' : ''}`;
+  const cpuValue = vCPUCount(cpu);
+  const memoryValue = humanizeMemory(memory);
+  const flavorValue = flavor ? toUIFlavorLabel(flavor) : null;
```

Comment:
```suggestion
  const flavorValue = flavor ? flavor : 'custom';
```

Reply:
If you get rid of the `toUIFlavorLabel` call, where are you going to put it?  It formats the name, so we can't omit it.